apps/aefate/rebar.config

```diff
@@ -0,0 +1,33 @@
+{minimum_otp_vsn, "20.1"}.
+
+{erl_opts, [debug_info]}.
+
+{escript_incl_apps, [aefate, aebytecode, lager, getopt]}.
+{escript_main_app, aefate}.
+{escript_name, aefate}.
```

Comment:
Is there a smoke test for this, hooked to CI?

Reply:
No. The stand alone engine is only used for debugging now. We might use it for some tests in the future and then this build will be tested.